cmd/cluster-agent/api/v1/language_detection.go

```diff
@@ -25,6 +26,12 @@ func InstallLanguageDetectionEndpoints(r *mux.Router) {
 }
 
 func postDetectedLanguages(w http.ResponseWriter, r *http.Request) {
+	if !config.Datadog.GetBool("language_detecion.enabled") {
```

Comment:
```suggestion
	if !config.Datadog.GetBool("language_detection.enabled") {
```
Also, I wonder if having a single parameter `language_detection.enabled` is the best move considering that the `language_detection` semantic does not cover `patching`. Moreover, I think it is better to keep language detection and patching independent considering that we might build features using language detection unrelated to this one in the future.

Perhaps it could be interesting to add a new parameter to control the patching logic.